### ISSUE TITLE
[Bug] Interpolation mode in object viewer cannot be changed.

### ISSUE THREAD

**Issue by adfriz** [leezer3/OpenBVE#1061]
### Description

as the title said
### Reproduction

i use fujikyu train from this website: https://charlesindobve.yolasite.com/japanese-railways.php

i just load the extension.cfg file to object viewer and press F8 to open the options window.

it seems the interpolation mode only can be changed once, and i cant see the difference when raising or lowering the anti aliasing level.

default, using anisotropic.
![image](https://github.com/user-attachments/assets/a6bca07b-30a3-46a7-a221-a97ef8ed3588)
i change it to nearest Neighbour
![image](https://github.com/user-attachments/assets/a5010c95-8755-46e2-a9a1-b912ef18d5e8)
i change it back to anisotropic but still looks the same as nearest neighbour
![image](https://github.com/user-attachments/assets/f1d68e5e-9a5b-4ca8-9fee-02cc247599ee)

### Related information 

* Operating system : windows 10
* OpenBVE v1.11.2 64 bit exe

**Comment by leezer3** [comment]
Interpolation must be as a result of the texture caching changes to speed up reloads. That should be easy enough to sort, although I'm not sure of the value of it.

Antialising has always required a restart of the program to change.
I don't think anyone was really expected to be trying to play with these with an object loaded; They really should be set and forget options.

**Comment by ginga81** [comment]
'Antialising has always required a restart of the program to change.'
Does that mean that each OpenBVE and ObjectViewer need to be restarted when we change Antialising?
If so, I didn't know that...
I think it want to write that somewhere in the help settings or something.

**Comment by leezer3** [comment]
I've now altered the interpolation option to clear the loaded when changed.
To note, this will produce a *new* lag spike when the options window is closed in this specific case, as it'll load all textures in the scene from scratch.
Previous to the texture caching on reload additions, this lag spike would have been at the reload point. 

---

With the main game specifically, not unless you're using the GL menu, but the viewers need a restart to change antialiasing.

TLDR:
The (openGL) window is created with a context that specifies the level of antialiasing the graphics drivers apply.
This context is fixed for the life of the window.

**Comment by adfriz** [comment]
Very late report.

Tested on daily builds 2025-10-23.

Works as expected
